api/overview/azure/latest/resourcemanager.devtestlabs-readme.md

```diff
@@ -1,12 +1,13 @@
 ---
-title: 
+title: DevTest Labs management client library for .NET
+description: Learn how to use the management client library for .NET to create labs in Azure DevTest Labs.
 keywords: Azure, dotnet, SDK, API, Azure.ResourceManager.DevTestLabs, devtestlabs
 author: pallavit
 ms.author: pallavit
 ms.date: 02/16/2023
 ms.topic: reference
 ms.devlang: dotnet
-ms.service: devtestlabs
+ms.service: devtest-lab
```

Comment:
This is a package-level overview (filename ends in `-readme.md`) and its metadata is generated by automation. This metadata will be overwritten on the next release of this package. To make this change persist you also need to open a PR to the metadata CSV file here: https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv

Set the value of `MSDocService` -- https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/602/Package-Onboarding-step-by-step?anchor=docs-metadata

Reply:
Updated in https://github.com/Azure/azure-sdk/pull/6231/files